data/presets/amenity/recycling/container/glass_bottles

```diff
@@ -0,0 +1,35 @@
+{
+    "icon": "maki-recycling",
+    "fields": [
+        "{amenity/recycling_container}"
+    ],
+    "moreFields": [
+        "{amenity/recycling_container}"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
```

Comment:
```suggestion
    "geometry": [
        "point"
    ],
```

The Wiki page for recycling container lists points and areas, but for this more specific preset of just glass bottle container, I suggest we start by just using points. This is more in line with the mapping practice I am aware of and with the numbers:
![image](https://user-images.githubusercontent.com/111561/146682199-a09a64a3-41b4-4bd8-9110-faca265c0a98.png)

https://taginfo.openstreetmap.org/tags/recycling%3Aglass_bottles=yes

Reply:
Good catch, limited to "point"